src/eset.c

```diff
@@ -165,6 +237,16 @@ eset_fit_alignment(eset_t *eset, size_t min_size, size_t max_size,
         pszind_t pind = sz_psz2ind(sz_psz_quantize_ceil(min_size));
         pszind_t pind_max = sz_psz2ind(sz_psz_quantize_ceil(max_size));
 
+	pszind_t pind_in = sz_psz2ind(sz_psz_quantize_floor(min_size));
+	if (sz_limit_usize_gap_enabled() && pind != pind_in) {
```

Comment:
This section need comments explaining why

Reply:
Added detailed explanations in `eset_first_fit`.